src/eventail/tmp_store.py

```diff
@@ -200,5 +209,5 @@ def peek_or_create(
             val = self.peek(key)
             if val is None:
                 val = factory()
-                self.set(key, val)
+                self.set(key, val, ttl)
```

Comment:
We should use the `ttl` from the function or from the class:
```suggestion
                self.set(key, val, ttl or self.ttl)
```

Reply:
that's already in the `set` function :wink: (it's ours)